Cargo.lock

```diff
@@ -5939,7 +5939,7 @@ dependencies = [
 
 [[package]]
 name = "spl-concurrent-merkle-tree"
-version = "0.1.2"
+version = "0.1.3"
```

Comment:
If this change needs doing, let's do it separately. I'm guessing it's an errant rebase, though.

Reply:
Yep, fixed!